is-palindrome.js

```diff
@@ -0,0 +1,15 @@
+// Napisz funkcję, isPalindrome(), która sprawdzi, czy podane słowo jest palindromem.
+
+function isPalindrome(string) {
+    const palind = string.split('').reverse().join('');
+
+    if (palind == string) {
```

Comment:
You shouldn't use double `=`. It's a bad practice. Check what's the best practice for comparing.